grouped_gemm_dev/moe/cutlass_kernels/moe_gemm/moe_permute_kernels.h

```diff
@@ -79,9 +79,12 @@ void moe_permute_kernel_launcher(
     const int num_cols,
     cudaStream_t stream)
 {
-    if (num_cols & 0x7 != 0)
-        throw std::runtime_error("num_cols of input activations must be multiples of 8.");
+    if ((kElementsPerAccess & (kElementsPerAccess - 1)) != 0)
```

Comment:
`kElementsPerAccess` here is not a changable argument for the user. It's for the data access inside the kernel. So, I think the conditional statement here is needless.

Reply:
Good point, how about using static_assert? 
`static_assert((kElementsPerAccess & (kElementsPerAccess - 1)) == 0, "kElementsPerAccess must be a power of 2.");`

Reply:
I mean you can just remove it, since it must be 16 for fp8, 8 for fp16/bf/16 and 4 for fp32. It's not changable.

Reply:
Okay, removed.